x-pack/plugins/alerting/server/rules_client/methods/get_alert_summary.ts

```diff
@@ -56,6 +56,8 @@ export async function getAlertSummary(
           start: parsedDateStart.toISOString(),
           sort: [{ sort_field: '@timestamp', sort_order: 'desc' }],
           end: dateNow.toISOString(),
+          // filter out execute-action event logs
+          filter: 'NOT event.action: execute-action',
```

Comment:
I am not sure but should we filter by `event.action : active-instance`? 
cc: @ymao1

Reply:
I think we look at the `execute` action as well during the post processing. We could probably add `event.provider: alerting` to the filter as well though.

Reply:
The summary log that was showing up has`event.provider: alerting`. Maybe we don't need a filter if I have changed the summary logs to not have an `instance_id`

Reply:
I meant in addition to the NOT filter you have here but if it is already filtered to alerting that is fine

Reply:
I added `event.provider: alerting`here https://github.com/elastic/kibana/pull/150088/commits/351bf861ddb404c87bb7369351d7cd3ce533adbd